react/Installments.tsx

```diff
@@ -45,22 +56,27 @@ function Installments({
     return null
   }
 
-  let [maxInstallmentOption] = commercialOffer.Installments
+  let [installmentsOption] = commercialOffer.Installments
+
+  if (installmentsCriteria === 'max-quantity') {
+    installmentsOption = pickMaxInstallmentsOption(
+      commercialOffer.Installments,
+      installmentOptionsFilter
+    )
+  }
 
-  commercialOffer.Installments.forEach(installmentOption => {
-    if (
-      installmentOption.NumberOfInstallments >
-      maxInstallmentOption.NumberOfInstallments
-    ) {
-      maxInstallmentOption = installmentOption
-    }
-  })
+  if (installmentsCriteria === 'max-quantity-no-interest') {
+    installmentsOption = pickMaxInstallmentsOptionWithNoInterest(
+      commercialOffer.Installments,
+      installmentOptionsFilter
+    )
+  }
```

Comment:
Nice! I also thought about using a switch-case here. Ended up using these `if` statements because it looked more readable to me for some reason 😅 .

I gotta say that having a `switch-case` with only 1 case looks weird to me, but it helps readability I'm all for it 😄

Reply:
Yeah! That's kind of subjective in this case, I thought it would help readability because of the default case